packages/modeling/src/primitives/star.js

```diff
@@ -12,7 +12,8 @@ const getRadiusRatio = (vertices, density) => {
   return 0
 }
 
-const getPoints = (vertices, radius, startAngle, center) => {
+const getPoints = (vertices, radius, startAngle) => {
```

Comment:
Please revert this change as getPoints() is an internal routine. The center is used when creating the 2D points.

Reply:
Done, this did fix the broken test as well.